autoload/vista/cursor.vim

```diff
@@ -391,7 +391,8 @@ function! vista#cursor#FoldOrJump() abort
     return
   endif
 
-  call vista#jump#TagLine(get(s:, 'cur_tag', ''))
```

Comment:
Shoudl also remove the other `s:cur_tag` if it's not used any more.

Reply:
Ok, removed.